common/client-libs/validator-client/Cargo.toml

```diff
@@ -90,4 +90,6 @@ default = ["http-client"]
 http-client = ["cosmrs/rpc"]
 generate-ts = []
 contract-testing = ["nym-mixnet-contract-common/contract-testing"]
-
+# Features below are added to make clippy happy, it seems like they're unused we should remove them
+tendermint-rpc-http-client = ["tendermint-rpc/http-client"]
```

Comment:
but I'm not 100% sure if adding those features will break the wasm client build. I don't remember if wasm didn't like the types or just the actual client (when used)